opendut-carl/src/cluster/manager.rs

```diff
@@ -94,11 +98,16 @@ impl ClusterManager {
             log::debug!("VPN disabled. Not creating VPN group.")
         }
 
+        let port_start = self.options.can_server_port_range_start;
+        let port_end = self.options.can_server_port_range_start + u16::try_from(member_interface_mapping.len()).unwrap();
```

Comment:
Avoid unwrap() whenever possible. E.g. or create 
```suggestion
        let port_end = self.options.can_server_port_range_start + u16::try_from(member_interface_mapping.len())
            .map_err(|cause| DeployClusterError::Internal { cluster_id, cause: cause.to_string() })?;
```